docs/pages/docs/config/config.md

```diff
@@ -32,6 +32,14 @@ We will cover each of these options below.
 The configuration object has a TypeScript type of `KeystoneConfig`, which can be imported from `@keystone-6/core/types`.
 This type definition should be considered the source of truth for the available configuration options.
 
+Note: It is important to pass a `TypeInfo` type argument to the config function as it ensures proper typing for the [Keystone Context](../context/overview.md). This type is automatically created in `.keystone/types`. You can customize the output path of the generated type by specifying it in the config object.
```

Comment:
@dcousens ... I see the website build failed... the context link should be `../context/overview` .... my bad

Reply:
@PaulAroo a fix-up pull request would be greatly appreciated! Don't stress :yellow_heart:

Reply:
@dcousens [#9025](https://github.com/keystonejs/keystone/pull/9025)